test/common/sycl_utils.hpp

```diff
@@ -0,0 +1,43 @@
+/***************************************************************************
```

Comment:
I'm going to complain about the file name. Whenever we have a file called *utils* it just ends up as a dumping ground for homeless or shared functionality where someone can't be bothered to think up a name for a new file. Can I suggest *usm_helpers.hpp* instead.

Reply:
In my next PR I was planning to move some of the content from `device_context.hpp` to this file which are not related to USM. This would be to move the switch to get the device type as a string and get the supported subgroup sizes as a string. They fit well enough in the same file IMO.

Reply:
I'll push back when someone starts putting things here that I don't think belong!